bindings/go/unicorn/unicorn_test.go

```diff
@@ -24,3 +24,16 @@ func TestMemUnmap(t *testing.T) {
 		t.Fatal(fmt.Errorf("Expected ERR_WRITE_UNMAPPED, got: %v", err))
 	}
 }
+
+func TestDoubleClose(t *testing.T) {
+	mu, err := NewUnicorn(ARCH_X86, MODE_32)
```

Comment:
IMHO this function name `NewUnicorn` looks weird. is it better to be just `Unicorn`?